content/r/concepts/dataframes/dataframes.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'Data Frames'
+Description: 'Data Frames are objects that store data into two dimensions.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Data Types'
+  - 'Objects'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+**Data Frames** are objects that store data into a table with two dimensions. The two dimensions are represented by columns and rows. The columns are the different variables and the rows are the observations of each variable. Data frames are useful for storing data of different types.
```

Comment:
Did you mean to refer to data types, or did you just mean different kinds of data? If the former, you should link out to the [data types](https://www.codecademy.com/resources/docs/r/data-types) entry.